modin/pandas/test/test_io.py

```diff
@@ -794,75 +802,64 @@ def test_read_csv_error_handling(self, on_bad_lines):
             on_bad_lines=on_bad_lines,
         )
 
+    @pytest.mark.parametrize("float_precision", [None, "high", "legacy", "round_trip"])
+    def test_python_engine_float_precision_except(self, float_precision):
```

Comment:
Can you elaborate on why python engine only?

Reply:
We already tested python engine in `test_read_csv_internal`, but I moved it to a separate test. Reason: https://github.com/modin-project/modin/pull/6997#discussion_r1511122976